src/amplitude/event.py

```diff
@@ -1,3 +1,17 @@
+"""Amplitude event module with Classes of events and special events wrappers.
+
+Classes:
+    Plan: Tracking plan info includes branch, source, version, version_id.
```

Comment:
I think Plan would make sense to go at the end of the list since most users wont need it. It would be good to keep the more heavily used classes at the top.

Reply:
Moved to the bottom